### ISSUE TITLE
Already a s.goel.im url.

### ISSUE THREAD

**Issue by rjain11** [karan/s#11]
While testing this I encountered something that can be changed and lead to more optimization. 
You could have a check if the url entered is already a s.goel.im url? 

Because when I tried it, I could enter an already shortened url and it would assign it another s.goel.im shortened url. 

Its a great hack though! Cheers!

**Comment by karan** [comment]
Yes. So the idea of `s` is to have a "template" for a really-fast URL shortener that anyone can get started. Checking for an existing URL would actually take more time than just making one. But yes, it's a totally valid thing to do if everything is on disk and not in memory.

Let me know if you decide to work on it.
